pkgs/applications/editors/emacs/sources.nix

```diff
@@ -34,55 +35,76 @@ let
     platforms = lib.platforms.all;
     mainProgram = "emacs";
   };
+
+  macportMetaFor = version: rev: {
+    homepage = "https://bitbucket.org/mituharu/emacs-mac/";
+    description = (mainlineMetaFor version).description + " - macport variant";
+    longDescription = (mainlineMetaFor version).longDescription + ''
+
+      This release is built from Mitsuharu Yamamoto's patched source code
+      tailored for MacOS X.
```

Comment:
```suggestion
      tailored for macOS.
```

Reply:
Is there any difference?

Reply:
Not really. It's just that Apple doesn't really use the term OSX anymore and it's also simply not version 10 anymore (OSXIII would sound kinda stupid).

I don't think we need to discuss this however.

Reply:
(I thought the X in OS X was a reference on Unix(TM).)